### PR TITLE
RHEL 9 STIG Fixes - Round Two

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

Adding `srg_requirement` to rules and other clean-ups.

#### Rationale:

RHEL9 STIG

### CODE REVIEW

File: shared/macros/ocil.jinja [hunk 1]
```diff
@@ -841,7 +841,7 @@ JINJA MACRO ERROR - Unknown init system '{{{ init_system }}}'.
 
 #}}
 {{%- macro ocil_clause_file_permissions(file, perms) -%}}
-    {{{ file }}} has unix mode {{{ perms }}}
+    {{{ file }}} does not unix mode {{{ perms }}}
```
Comment (ggbecker) [inline shared/macros/ocil.jinja:844]:
```suggestion
    {{{ file }}} does not have unix mode {{{ perms }}}
```

File: controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml [hunk 2]
```diff
@@ -9,8 +9,6 @@ controls:
             - kerberos_disable_no_keytab
             - accounts_password_all_shadowed_sha512
             - package_rsyslog-gnutls_installed
-            - rsyslog_remote_tls
-            - package_libreswan_installed
+            - libreswan_approved_tunnels
             - configure_kerberos_crypto_policy
-            - package_krb5-workstation_removed
```
Comment (ggbecker) [inline controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml:15]: I see that in RHEL8 STIG this rule is still there with this SRG. What is the reason behind this change?

Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml:15]: Recent versions of the package appear to use FIPS-compliant cryptographic modules. See the latest
RHEL8 STIG (RHEL-08-010162) for more details.

Comment (ggbecker) [reply controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml:15]: makes sense, maybe we will need to provide documentation that krb5 can use the FIPS stuff at some point

Comment (Mab879) [reply controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml:15]: Possibly, I was trying to find some docs for it. I think we need, in general, to find docs (or something) on the status of FIPS and krb5 in RHEL9. Do we want to merge this now and raise an issue to figure out what is going on?

Comment (ggbecker) [reply controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml:15]: Let's merge this one now and I will create an issue to track this.

**Review by ggbecker — APPROVED** [review]
